band/www/home.py

```diff
@@ -1,17 +1,31 @@
 from __future__ import unicode_literals
 import frappe
+from frappe.model.document import Document
 
 def get_context(context):
-    carousel_items = [
-        {'num':0, 'title': 'Chicago','image_src':'/assets/band/images/chicago.jpg','caption':'Thank you, Chicago - A night we won\'t forget.', 'active':'active'},
-        {'num':1, 'title': 'Los Angeles','image_src':'/assets/band/images/la.jpg','caption':'Even though the traffic was a mess, we had the best time playing at Venice Beach!'},
-        {'num':2, 'title': 'New York','image_src':'/assets/band/images/ny.jpg','caption':'The atmosphere in New York is lorem ipsum.' }
-    ]
+	
+    carasoul =  frappe.get_list("Carousel", fields=["title", "image", "caption"])
+    carousel_items=[]
+    active = 'active'
+    for counter, r in enumerate(carasoul):
+        if counter <> 0:
+            active = ''
```

Comment:
if we need active you should add it as a field in doctype and handle it in front end